docs/new-architecture-app-modules-android.md

```diff
@@ -272,8 +363,78 @@ public class MyApplication extends Application implements ReactApplication {
                 return new MyApplicationTurboModuleManagerDelegate.Builder();
             }
         };
+}
+```
+
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+class MyApplication() : Application(), ReactApplication {
+    private val mReactNativeHost: ReactNativeHost =
+        object : ReactNativeHost(this) {
+            /* ... */
+            val useDeveloperSupport: Boolean
+                get() {
+                    /* ... */
+                }
+
+            val packages: List<Any>
+                protected get() {
+                    val packages: MutableList<ReactPackage> = PackageList(this).getPackages()
+
+                    // Add those lines
+                    packages.add(
+                        object : TurboReactPackage() {
+                            @Nullable
+                            fun getModule(
+                                name: String,
+                                reactContext: ReactApplicationContext?
+                            ): NativeModule? {
+                                if ((name == NativeAwesomeManager.NAME)) {
+                                    return NativeAwesomeManager(reactContext)
+                                } else {
+                                    return null
+                                }
+                            }
+
+                            val reactModuleInfoProvider: ReactModuleInfoProvider
+                                get() = ReactModuleInfoProvider {
+                                    val moduleInfos: MutableMap<String, ReactModuleInfo> = HashMap()
+                                    moduleInfos.put(
+                                        NativeAwesomeManager.NAME,
+                                        ReactModuleInfo(
+                                            NativeAwesomeManager.NAME,
+                                            "NativeAwesomeManager",
+                                            false, // canOverrideExistingModule
+                                            false, // needsEagerInit
+                                            true, // hasConstants
+                                            false, // isCxxModule
+                                            true // isTurboModule
+                                        )
+                                    )
+                                    moduleInfos
+                                }
+                        }
+                    )
+                    return packages
+                }
+
+            protected fun getJSMainModuleName(): String {
+                /* ... */
+            }
+
+            protected fun getReactPackageTurboModuleManagerDelegateBuilder():
```

Comment:
Here and in other places you're missing the `override` key

Reply:
added override key to the function declaration